core/src/main/scala/cromwell/util/JsonEditor.scala

```diff
@@ -225,25 +244,29 @@ object JsonEditor {
       def filterCallEntry(json: Json): ErrorOr[JsonObject] = {
         for {
           callObject <- json.asObject.map(_.validNel).getOrElse(s"Call entry unexpectedly not an object: $json".invalidNel)
+          // Filter the call object itself, no recursing for subworkflows.
           shallowFilteredCallObject = filterGroup.applyIncludes(callObject)
           subworkflow = callObject(Keys.subWorkflowMetadata).getOrElse(Json.fromJsonObject(JsonObject.empty))
-          shallowFilteredSubworkflow <- applyIncludes(subworkflow)(filterGroup)
-          shallowFilteredSubworkflowObject <- shallowFilteredSubworkflow.asObject.map(_.validNel).getOrElse(s"subworkflow unexpectedly not an object: $subworkflow ".invalidNel)
-          builtCallObject = buildCallObject(callObject, shallowFilteredCallObject, shallowFilteredSubworkflowObject)
+          filteredSubworkflow <- applyIncludes(subworkflow)(filterGroup)
+          filteredSubworkflowObject <- filteredSubworkflow.asObject.map(_.validNel).getOrElse(s"subworkflow unexpectedly not an object: $subworkflow ".invalidNel)
+          builtCallObject = buildCallObject(callObject(Keys.shardIndex), callObject(Keys.attempt), shallowFilteredCallObject, filteredSubworkflowObject)
```

Comment:
Does this fail gracefully if the `shardIndex` or `attempt` are missing from the json in GCS?

(an error response would be fine since this is a "should never happen in the real world" situation, but a system-wide Cromwell crash is probably not)